test/format/config.c

```diff
@@ -301,7 +301,7 @@ config_compression(const char *conf_name)
 		break;
 #endif
 #ifdef HAVE_BUILTIN_EXTENSION_ZSTD
-	case 15: case 16 case 17:		/* 15% zstd */
```

Comment:
This is a bug that only shows up when building with `builtins=zstd`, which we don't yet have automated.